woocommerce/woocommerce
# Update coming soon page link to template editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/46562.

- Update coming soon page link to directs merchants toward the template's editable screen in the Site Editor.


![Image](https://github.com/woocommerce/woocommerce/assets/9312929/efa2d5bd-fdae-4ab6-b661-c08fbc40887f)

![Image](https://github.com/woocommerce/woocommerce/assets/9312929/913a30ef-1af3-4bae-a534-5641efcdb955)





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Open a new window and enable `launch-your-store` feature flag.
3. Go to `WooCommerce > Settings`
4. Go to `Site visibility` settings.
5. Click on the coming soon page editor link. (`Your site is hidden from visitors behind a “Coming soon” landing page until it’s ready for viewing. You can customize your “Coming soon” landing page via the **Editor**.`)
6. Verify that the link opens the coming soon template in the site editor.
7. Go to `WooCommerce > Home`
8. Click on the `Site coming soon` badge.
9. Click on the `Customize "Coming soon" page` menu item.
10. Verify that the link opens the coming soon template in the site editor.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
